curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-the-spread-operator-to-evaluate-arrays-in-place.english.md

```diff
@@ -43,7 +43,7 @@ Copy all contents of <code>arr1</code> into another array <code>arr2</code> usin
 ```yml
 tests:
   - text: <code>arr2</code> should be correct copy of <code>arr1</code>.
-    testString: assert(arr2.every((v, i) => v === arr1[i]));
+    testString: assert(arr2.every((v, i) => v === arr1[i]) && arr2.length > 0);
```

Comment:
```suggestion
    testString: assert(arr2.every((v, i) => v === arr1[i]) && arr2.length);
```

Because `0` is falsy, we can assert for the truthiness of the `arr2.length` value. 🙂

Reply:
That's right. Since I have to go back to my code months after writing it when a customer asks me to expand on it, I always go for the most explicit implementation unless there's a difference in performance - it makes picking it up again easier. It's gotten to the point where I do it without even thinking about it. Your correction is (of course) more in line with the style used in this project. Thanks!